test/unit/specs/Slides.spec.js

```diff
@@ -1,197 +1,210 @@
 import Vue from 'vue'
+import sinon from 'sinon'
+import Vuex from 'vuex'
+import VueRouter from 'vue-router'
 import Slides from '@/components/Slides'
+import slideModule from '@/store/modules/slide/index'
+import routerModule from '@/router/index'
```

Comment:
you don't need to specify `index`, webpack will auto-magically look for it

Reply:
Done